docs/NEW_LANGUAGE_FEATURE_INTEGRATION_PROCESS.md

```diff
@@ -0,0 +1,31 @@
+# Check Update Procedure
```

Comment:
IMO `Check Update Procedure` => `Check Update Procedure for New Language Features` or something similar.

You don't want to confuse people that this is for normal check updates like adding a new property. This is specifically if we change ANTLR. Even though the file name says the same thing, it just helps to be explicit in what the user is reading. Some will gloss over a file name, especially if it is long or hard to read.